Lesson_2/task_8.js

```diff
@@ -6,23 +6,32 @@
 
 
 const human = Object.create({}, {
-    fullName: {
-        get() {
-            return `${this.firstName} ${this.lastName}`
+        fullName: {
+            get() {
+                return `${this.firstName} ${this.lastName || ''}`
+            },
+            set(value) {
+                if (value.split(' ').length < 2) {
+                    this.firstName = value
+                    this.lastName = ''
+                } else {
+                    [this.firstName, this.lastName] = value.split(' ')
+                }
+            }
         },
```

Comment:
Вы можете оставить деструктуризацию строки в сеттере, и просто хранить `undefined`:
```suggestion
        fullName: {
            get() {
                return [this.firstName, this.lastName].filter(Boolean).join(' ')
            },
            set(value) {
                [this.firstName, this.lastName] = value.trim().split(' ').filter(Boolean)
            }
        },
```